en/debug/consoles.md

```diff
@@ -0,0 +1,43 @@
+# PX4 Consoles/Shells
+
+PX4 enables terminal access to the system through the [MAVLink Shell](../debug/mavlink_shell.md) and the [System Console](../debug/system_console.md).
+
+This page explains the main differences and how the console/shell are used.
+
+
+## System Console vs. Shells {#console_vs_shell}
+
+The PX4 *System Console* provides low-level access to the system, debug output and analysis of the system boot process.
+
+There is just one *System Console*, which runs on one specific UART (the debug port, as configured in NuttX), and is commonly attached to a computer via an FTDI cable (or some other debug adapter like  a [Dronecode probe](https://kb.zubax.com/display/MAINKB/Dronecode+Probe+documentation)).
+- Used for *low-level debugging/development*: bootup, NuttX, startup scripts, board bringup, development on central parts of PX4 (e.g. uORB).
+- In particular, is the only place where all boot output (including information about applications auto-started on boot) is printed.
+
+Shells provide higher-level access to the system:
+- Used for basic module testing/running commands.
+- Only display the output of modules you start (and therefore cannot debug the boot process).
```

Comment:
Partially: as long as the system boots, you can still use `dmesg` for debugging. In general `dmesg` is very powerful and brings the MAVLink shell much closer to the system shell (with `dmesg -f &` for example, you also see the output of background tasks).

Reply:
Thanks @bkueng. I re-wrote this - can you please check. Tried to capture that you can only "directly" see stuff you start, but dmesg allows indirect access to that.